README.md

```diff
@@ -295,6 +330,27 @@ For more information and to see an example of this in use take a look at the [Sp
 
 Instructions on how to run the Spring example project can be found in the [yoti-sdk-spring-boot-example](/yoti-sdk-spring-boot-example) folder.
 
+## Breaking changes and enhancements made in v2.0.0
+
```

Comment:
Since the README is already quite large, do you think it would be better to put these changes in a CHANGELOG.md (like we have in [.NET](https://github.com/getyoti/yoti-dotnet-sdk/blob/master/CHANGELOG.md), Python, Ruby & Node).

We can place a link to the changelog in the README still. This would mean the README wouldn't have to grow with the changes we make for each new version.

Reply:
I'm not sure.  If we're moving towards using the Yoti website as the primary home for documentation, then these READMEs will probably only have stuff like change logs in them.

Reply:
We also have these changes in the release notes on Github (e.g. [.NET](https://github.com/getyoti/yoti-dotnet-sdk/releases)). I'm just thinking that it's polluting the README a bit if we do this every time, when we can represent this information in other places

Reply:
That does look nice.  We haven't been using those Releases in Java SDK - we should fix that.
But I'm still concerned about how many places we want people to look - we have 4x READMEs here, as well as the Yoti website.